core/esmf-aspect-meta-model-python/tests/integration/resources/org.eclipse.esmf.test.entity/2.0.0/AspectWithEntity.ttl

```diff
@@ -44,3 +44,15 @@
    samm:preferredName "Entity Property"@en ;
    samm:description "This is a property for the test entity."@en ;
    samm:characteristic samm-c:Text .
+
+:AnotherAspect a samm:Aspect ;
+   samm:preferredName "Another test Aspect"@en ;
+   samm:description "This is a test description for the second Aspect"@en ;
+   samm:see <http://example.com/> ;
+   samm:properties ( :someStrangeProperty ) ;
+   samm:operations ( ) .
+
+:someStrangeProperty a samm:Property ;
+   samm:preferredName "Some weird Property"@en ;
+   samm:description "This is a property for the test entity 2."@en ;
```

Comment:
Can we change this so it doesn't sound as if the test code is confused? :D
```suggestion
:someProperty a samm:Property ;
   samm:preferredName "Some Property"@en ;
   samm:description "This is a property for the test entity 2."@en ;
```

Reply:
O, sorry
Just rename it, because could not find a reason of the issue.